src/startCronjobs.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+
+import {CronJob} from 'cron'
+// import moment from 'moment'
+import moment from 'moment-timezone'
+import Offer from './models/Offer'
+import forms from './models/forms'
+import UserAction, {
+  ACTION_AUTOMATICALLY_DELETED_EXPIRED,
+  ACTION_AUTOMATICALLY_DELETED_NOT_CONFIRMED
+} from './models/UserAction'
+
+const TIMEZONE = moment.tz.guess()
+
+const deleteOffer = async (offer: Offer) => {
+  const {FormModel} = forms[offer.city]
+  // Delete the form data
+  await FormModel.findByIdAndDelete(offer.formData._id).exec()
+  // Delete the offer
+  await Offer.findByIdAndDelete(offer._id).exec()
+}
+
+const deleteExpired = async () => {
+  const twoYearsAgo = moment().subtract(2, 'year')
```

Comment:
2 Years? Quite a long time.

Reply:
That's something we agreed on with the people from Neuburg I think. The thought was to have statistics for more than one year.